modules/ROOT/partials/configuration/icon_list.adoc

```diff
@@ -59,6 +59,7 @@
 | `+embed+` | image:icons/embed.svg[embed.svg] | `+embed.svg+`
 | `+emoji+` | image:icons/emoji.svg[emoji.svg] | `+emoji.svg+`
 | `+export+` | image:icons/export.svg[export.svg] | `+export.svg+`
+| `+export-word+` | image:icons/export-word.svg[export-word.svg] | `+export-word.svg+`
```

Comment:
@hamza0867 Can you let me know if the `.svg` will match this naming convention once its ready.